### PR TITLE
CoreTap.default_remote: Use Linuxbrew/core [Linux]

### PR DESCRIPTION
**Pull request by sjackman**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

### CODE REVIEW

File: Library/Homebrew/cmd/update.sh [hunk 1]
```diff
@@ -24,7 +24,12 @@ git() {
 
 git_init_if_necessary() {
   BREW_OFFICIAL_REMOTE="https://github.com/Homebrew/brew"
-  CORE_OFFICIAL_REMOTE="https://github.com/Homebrew/homebrew-core"
+  if [[ -n "$HOMEBREW_MACOS" ]] || [[ -n "$HOMEBREW_FORCE_HOMEBREW_ORG" ]]
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/update.sh]: This is just used by `brew test-bot`, right?

File: Library/Homebrew/extend/os/linux/tap.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class CoreTap < Tap
+  def default_remote
+    if ENV["HOMEBREW_FORCE_HOMEBREW_ORG"]
+      "https://github.com/Homebrew/homebrew-core".freeze
+    else
+      "https://github.com/Linuxbrew/homebrew-core".freeze
```
Comment (MikeMcQuaid) [inline Library/Homebrew/extend/os/linux/tap.rb]: Will this direct people to file Linux formulae issues there (if we don't already do that)?

**Comment by MikeMcQuaid** [comment]
May also wish/need to adjust https://github.com/Homebrew/homebrew-test-bot/blob/master/cmd/brew-test-bot.rb

File: Library/Homebrew/cmd/update.sh (hunk 1 above)
Comment (sjackman) [reply Library/Homebrew/cmd/update.sh]: `HOMEBREW_FORCE_HOMEBREW_ORG` you mean? Yes, I believe so.
See https://github.com/Homebrew/homebrew-test-bot/blob/ac511e47217a49dfbeffcb2f09e384f8aa49e13b/cmd/brew-test-bot.rb#L1325-L1329

File: Library/Homebrew/extend/os/linux/tap.rb (hunk 2 above)
Comment (sjackman) [reply Library/Homebrew/extend/os/linux/tap.rb]: The user is directed to `OS::ISSUES_URL`, which on Linux is https://github.com/Linuxbrew/brew/wiki/Troubleshooting.
As far as I can tell, `CoreTap.instance.issues_url` is not used when `official_tap?` is true.

**Comment by sjackman** [comment]
> May also wish/need to adjust https://github.com/Homebrew/homebrew-test-bot/blob/master/cmd/brew-test-bot.rb

True. Linuxbrew is using our fork Linuxbrew/test-bot. We'll want to merge that into Homebrew/test-bot. That can come after merging this PR though.

**Pull request merged**